modules/templates/proc-adding-templates.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-adding-templates_{context}"]
+= Adding an existing template to {product} by using the Template Editor
+
+You can add an existing template to your {product} instance by using the Template Editor.
+
+.Prerequisites
+
+* You have created a directory that contains template files.
+* You have created at least one template YAML file in the directory.
+
+.Procedure
+
+. Access the Template Editor by using one of the following options:
++
+image::rhdh/template-editor.png[Template Editor]
+** Open the URL `\https://<rhdh_url>/create/edit` for your {product} instance.
+** Click *Create...* in the navigation menu of the {product} console, then click the overflow menu button and select *Template editor*.
+. Click *Load Template Directory*. This allows you to upload a template from a local directory, by navigating to the directory and clicking *Select*.
+. A *Let site view files?* pop-up appears. Click *View files*.
+. In the *Name ** field, enter a unique name for your template.
+. From the *Owner* drop-down menu, choose an owner for the template.
+. Click *Next*.
+. In the *Repository Location* view, enter information about the the hosted repository to publish the template to:
+.. Select an available *Host* from the drop-down menu.
++
+--
+[NOTE]
```

Comment:
Same remark as before. You assume that the user will load a template containing an action to publish on github

Reply:
See above comment - let's have QE verify whether there are additional details to include here for other repository types. I think from a user perspective it's better to include the information that we have currently at least, since it's the same for github and gitlab.